lib/plugins/aws/package/lib/merge-iam-templates.js

```diff
@@ -109,6 +109,11 @@ module.exports = {
     }-${this.provider.getStage()}`;
     const logGroupsPrefix = this.provider.naming.getLogGroupName(canonicalFunctionNamePrefix);
 
+    const assumeRolePolicyDocumentStatements =
```

Comment:
Let's do these changes in the context of the event module (as e.g. we do in the case of `cloudFront` implementation -> https://github.com/serverless/serverless/blob/a6bcc62182b6fe974d12d547c00e560882c55e05/lib/plugins/aws/package/compile/events/cloud-front.js#L614-L627)

Reply:
done